recipes/cmip7-repack/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set python_min = "3.11" %}
+{% set name = "cmip7-repack" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/c/cmip7_repack/cmip7_repack-{{ version }}.tar.gz
+  sha256: b04463487160053d9c9603bed4a62fd72e4c20f29652442a0c06ff004e84c7d6
+
+build:
+  noarch: python
```

Comment:
This one has scripts and fails on Windows. You must do a OS-Noarch to prevent it from being installed on Windows. See https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies

Reply:
Brill! Thanks, will do tomorrow, and re-ping you, sorry, end of day here 🙂